src/main/java/ponyo/commands/AddCommand.java

```diff
@@ -0,0 +1,99 @@
+package ponyo.commands;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import ponyo.data.task.Deadline;
+import ponyo.data.task.Event;
+import ponyo.data.task.Task;
+import ponyo.data.task.TaskList;
+import ponyo.data.task.Todo;
+import ponyo.storage.Storage;
+import ponyo.ui.Ui;
+
+/**
+ * Adds a new task to the task list.
+ */
+public class AddCommand extends Command {
```

Comment:
Would it be better if the class was named CommandAdder? I think AddCommand might sound like a method name :)